recipes/backward-cpp/all/conanfile.py

```diff
@@ -36,16 +36,23 @@ def _has_stack_walking(self, type):
     def _has_stack_details(self, type):
         return self.options.stack_details == type
     
+    def _supported_os(self):
+        return ["Linux", "Macos", "Android", "Windows"] if tools.Version(self.version) < "1.5" \
+               else ["Linux", "Macos", "Android"]
```

Comment:
Am I reading this right that before 1.5 Windows is supported, but since 1.5 it is not? Shouldn't it be reversed?

Reply:
Yes It's a mistake :-)